setup.py

```diff
@@ -42,6 +42,8 @@
         install_requires=[
         'setuptools',
         "plone.volto",
+        'plone.base',
```

Comment:
@nileshgulia1 Do we really need this dependency here?

Reply:
Fixed in [4b050a6](https://github.com/eea/eea.volto.policy/pull/14/commits/4b050a67b2ab6787f9567f309c7329d2e9dde11f) since we don't need this PLone 6 and already patched interfaces for Plone 5